components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactNode, ReactElement, CSSProperties, Component, cloneElement } from 'react'
+import BreadcrumbItem, { BreadcrumbItemProps } from './BreadcrumbItem'
+
+export interface BreadcrumbProps {
+  /** crumbs of the data to be shown */
+  crumbs?: string[] | ReactNode[];
+  /** separate to be using between crumbs */
+  separator?: string | ReactNode;
+  /** styles specified by the user */
+  style?: CSSProperties;
+  /** Breadcrumb Items in children */
+  children?: ReactNode[],
+  /** className for user to specify their class */
+  className?: string
+}
+
+export default class Breadcrumb extends Component<BreadcrumbProps, any> {
```

Comment:
Would be totally cleaner, however linter suggested to convert it to class since we're using a property `.Item` on this component which needs it to be a class I believe. WDYT are there any workarounds?